src/exercises/model.py

```diff
@@ -0,0 +1,39 @@
+from collections import namedtuple
+from pathlib import Path
+from typing import List
+
+
+class Exercise(namedtuple("Exercise", ["name"])):
+    pass
+
+
+class Directory(namedtuple("Directory", ["name", "children"])):
+    def list_exercises(self) -> List[Path]:
+        return self.__inner_list_exercises__("")
+
+    def __inner_list_exercises__(self, prefix) -> List[Path]:
```

Comment:
this `__method__` convention is for standard Python function, what you want here is a private nomenclature, which is `_method`

Reply:
it trigger `src/exercises/model.py:30:23: W0212: Access to a protected member _inner_list_exercises of a client class (protected-access)`

So I made them regular methods